io.typora.editor.json

```diff
@@ -0,0 +1,74 @@
+{
+  "id": "io.typora.editor",
```

Comment:
I'd probably go with `io.typora.Typora` but I don't have strong opinion on that.